SportsHub/SportsHub.Core/Models/Article.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace SportsHub.Domain.Models
+{
+    public class Article
+    {
+        public Article()
+        {
+            Comments = new HashSet<Comment>();
```

Comment:
why HashSet is here?

Reply:
To make sure we do not get the same comment more than once

Reply:
It is the same [here](https://github.com/mehalyna/Sf-069.Net-Project/blob/17fe27ce3020892f587503a964de1e0414d2c04e/SportsHub/SportsHub.Core/Models/Role.cs#L7) and the pull request was approved. This is the way I know collections are initialized in EF.